notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -0,0 +1,1296 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #436.    )
</pre><p>Remove this parenthesis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>